sdks/python/apache_beam/runners/interactive/interactive_runner.py

```diff
@@ -220,7 +229,7 @@ def visit_transform(self, transform_node):
 
   # TODO(victorhc): Move this method somewhere else if performance is impacted
   # by generating a cluster during runtime.
-  def _create_dataproc_cluster_if_applicable(self, user_pipeline):
+  def _get_dataproc_cluster_master_url_if_applicable(self, user_pipeline):
```

Comment:
Please add the type hints.

Reply:
Good call, I've now added type hints here. Thanks!